crates/re_data_ui/src/image.rs

```diff
@@ -84,7 +84,10 @@ fn tensor_ui(
     let meter = if meaning == TensorDataMeaning::Depth {
         ctx.entity_db
             .store()
-            .query_latest_component::<DepthMeter>(entity_path, &ctx.current_query())
+            .query_latest_component::<DepthMeter>(
+                entity_path,
+                &ctx.current_query_for_entity_path(entity_path),
```

Comment:
Will a tensor entity ever be on the blueprint timeline? I thought that was only applicable to stuff like containers and space view entities?

Reply:
I guess I'm a bit confused why some `current_query()` has become `current_query_for_entity_path`, but not all.

Reply:
> Will a tensor entity ever be on the blueprint timeline?

Yes -- data overrides will be stored in the blueprint. So if you set an override, and then select the blueprint entity representing that override you'll hit this code path.

> I guess I'm a bit confused why some current_query() has become current_query_for_entity_path, but not all.

I scoped my search & replace to the data_ui elements which is the only context in which it should theoretically matter .